og.module

```diff
@@ -86,6 +86,16 @@ define('OG_MEMBERSHIP_REQUEST_FIELD', 'og_membership_request');
  */
 
 /**
+* Implements hook_entity_field_storage_info_alter().
+*/
+function og_entity_field_storage_info_alter(&$fields, \Drupal\Core\Entity\EntityTypeInterface $entity_type) {
+  if (!empty($fields[OG_AUDIENCE_FIELD])) {
+    // Decorate the audience field instance to set the custom storage.
+    $fields[OG_AUDIENCE_FIELD] = new \Drupal\og\Field\FieldStorageConfig($fields[OG_AUDIENCE_FIELD]);
```

Comment:
Yes, this is basically a wrapper around the core `FieldStorageConfig` class. I agree, I have been talking to people about potentially the best way to fix this in core, if possible. I will definitely work on that.

However, I think we need to go with this for now, with the hope we can remove or simplify later. This keeps our functionality in a working state. Would you be OK with that?